tests/sentry/api/endpoints/test_organization_releases.py

```diff
@@ -1343,3 +1346,156 @@ def test_commits_with_patch_set(self):
         self.assert_file_change(file_changes[1], 'A', 'templates/goodbye.html', commits[1].id)
         self.assert_file_change(file_changes[2], 'M', 'templates/hello.html', commits[1].id)
         self.assert_file_change(file_changes[3], 'D', 'templates/hola.html', commits[0].id)
+
+
+class ReleaseSerializerWithProjectsTest(TestCase):
```

Comment:
Do you need database support in these tests? If not you can inherit from `unittest.TestCase` and get much faster execution time.

Reply:
Hmm... by database support does that include lookup?  I'll play with it for a bit. I know when the serializer has a`UserField` it gets the user from the database based on the username. That's not all of them though. Let me try it and see what happens.